tests/dummy/app/templates/application.hbs

```diff
@@ -18,11 +18,21 @@
                   Dynamic behaviour
                 </AuNavigationLink>
               </li>
+              <li class="au-c-list-navigation__item">
+                <AuNavigationLink @route="form" @model="dynamic-behaviour-v2">
+                  Dynamic behaviour v2
```

Comment:
Nitpick, but maybe we should rename the old version to `Dynamic behavior (legacy)` with a disclaimer that we shouldn't be using that setup anymore?

This one could then simply be `Dynamic behavior` or `Conditional rendering` or something?